### ISSUE TITLE
name pointers with arbitrary binary key is not rendered on names endpoint

### ISSUE THREAD

**Issue by sborrazas** [aeternity/ae_mdw#986]
Name pointers will only diplay a single value right now. We should look into why this behaves like this https://github.com/aeternity/ae_mdw/blob/f6f69e3bdf90aa2547746d6e85aed0d3724cb49b/lib/ae_mdw/names.ex#L382

This might have been introduced here https://github.com/aeternity/ae_mdw/commit/ce9293bf582a7e799710f90ca4974f033dc45b84#diff-6509d83849ebb5bd9a1db2a6cf655650104e3f1d93a0a09d4f9a54b93f16eaa2R201

**Comment by jyeshe** [comment]
For names endpoint this is the actual reason 95b2f5af related to the comment:
```
# For some reason we're only grabbing the last pointer to build a hash that
# contains "account_pubkey" as the only key.
```

**Comment by jyeshe** [comment]
Already returning pointers on:

- `/v2/txs?type=name_update_tx
- `/v2/names/:name`

Missing for `/v2/names` fixed by #995

**Comment by jyeshe** [comment]
Already returning pointers on:

- `/v2/txs?type=name_update_tx
- `/v2/names/:name`

Missing for `/v2/names` fixed by #995

**Comment by jyeshe** [comment]
> Name pointers will only diplay a single value right now. We should look into why this behaves like this
> 
> https://github.com/aeternity/ae_mdw/blob/f6f69e3bdf90aa2547746d6e85aed0d3724cb49b/lib/ae_mdw/names.ex#L382
> 
> This might have been introduced here [ce9293b#diff-6509d83849ebb5bd9a1db2a6cf655650104e3f1d93a0a09d4f9a54b93f16eaa2R201](https://github.com/aeternity/ae_mdw/commit/ce9293bf582a7e799710f90ca4974f033dc45b84#diff-6509d83849ebb5bd9a1db2a6cf655650104e3f1d93a0a09d4f9a54b93f16eaa2R201)

It turns out that ce9293b was a revert because the previous commit to it had a rendering issue. Some non account pointer is not rendered properly:
https://testnet.aeternity.art/mdw/v2/txs?type=name_update&direction=forward&limit=20
